src/GitHub.App/Models/PullRequestModel.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Globalization;
+using GitHub.Primitives;
+using GitHub.VisualStudio.Helpers;
+using NullGuard;
+
+namespace GitHub.Models
+{
+    public sealed class PullRequestModel : NotificationAwareObject, IPullRequestModel
+    {
+        public PullRequestModel(int number, string title, IAccount author, DateTimeOffset createdAt, DateTimeOffset? updatedAt = null)
+        {
+            Number = number;
+            Title = title;
+            Author = author;
+            CreatedAt = createdAt;
+            UpdatedAt = updatedAt ?? CreatedAt;
+        }
+
+        public void CopyFrom(IPullRequestModel other)
```

Comment:
Well yes, but changing the name of the method in the interface because this implementation happens to only copy properties doesn't really feel logical. 😜